tests/test_examples_json.py

```diff
@@ -42,7 +52,11 @@ def get_data(self):
             return json.load(fp)
 
 
-class TestMultipleNetworks(BaseTest):
+class TestMultipleNetworks(BaseTestExampleDataValid):
+    """
+    Tests examples/json/multiple-networks.json is valid.
+    See
```

Comment:
Is there something else to be added here?

Reply:
Can't remember. Removed "See"